### ISSUE TITLE
GCM--Identity mapping

### ISSUE THREAD

**Issue by Klaviersonate** [ReaFly/ACSNet#1]
GCM模块的恒等映射分支，看了结构图和代码，请教下您，这里是先用了一个卷积再是non-local结构吗？卷积核尺寸是什么，在此处有什么作用呢？non-local的输出通道根据什么决定的呢？

**Comment by ReaFly** [comment]
您好，卷积核尺寸是1*1的，作用是降维，non-local模块的输出通道数与输入是一致的

**Comment by Klaviersonate** [comment]
> 您好，卷积核尺寸是1*1的，作用是降维，non-local模块的输出通道数与输入是一致的

non-local有bottleneck，也是降维，就是降维再升维减少参数量。那么1*1卷积的降维和non-local的降维有什么区别吗

**Comment by ReaFly** [comment]
1*1的降维是为了和其他三个分支的输出通道数保持一致，即经降维得到一个更为压缩的特征后再借助non-local捕获长距离依赖关系，non-local内部的降维再升维是降低该模块的计算量

**Comment by Klaviersonate** [comment]
您好，称为 恒等映射，是non-local包含一个短连接，某种角度看non-local和resnet引入短连接类似，都是 输出H(x)=F(x)+输入x， 这个意思吗

**Comment by ReaFly** [comment]
恒等映射更多的还是相对于其他三个分支所说的，因为该分支没有做pooling操作，只是借助non-local捕获长距离依赖关系
